### ISSUE TITLE
ctrl+E not reload extension but open search bar?

### ISSUE THREAD

**Issue by neilfranci** [wxt-dev/wxt#339]
### To Reproduce

```bash
pnpx wxt@latest init <project-name>
pnpm i
pnpm dev
```
[learn-wxt.zip](https://github.com/wxt-dev/wxt/files/13906303/learn-wxt.zip)

Steps to reproduce the bug using the reproduction:

1. Install dependencies: `pnpm i`
2. Start dev mode: `pnpm dev`
3. Press Ctrl+E on Windows it focus on the search bar but do nothing with the extension

```
  System:
    OS: Windows 11 10.0.22631
    CPU: (16) x64 12th Gen Intel(R) Core(TM) i5-12500H
    Memory: 3.82 GB / 15.73 GB
  Binaries:
    Node: 20.10.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.21 - C:\Program Files\nodejs\yarn.CMD
    npm: 10.2.3 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.14.1 - C:\Program Files\nodejs\pnpm.CMD
  Browsers:
    Edge: Chromium (120.0.2210.121)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    wxt: ^0.13.0 => 0.13.5
```

**Comment by aklinker1** [comment]
Hmm, `ctrl+E` is a keyboard shortcut on windows... Ok, we'll have to change it. 

Thoughts on `ctrl+shift+E`?

**Comment by neilfranci** [comment]
> Hmm, `ctrl+E` is a keyboard shortcut on windows... Ok, we'll have to change it.
> 
> Thoughts on `ctrl+shift+E`?

I don't think `ctrl+shift+E` would make it more complicated, rather click on `Extension reloader` add-ons on the store faster. Maybe `shift+E` or `shift+space+E`

**Comment by neilfranci** [comment]
I think `Alt | Ctrl` is required, maybe `Ctrl+B`?

**Comment by aklinker1** [comment]
On Firefox, `ctrl+B` opens bookmarks :/ So that's not a perfect solution either. Better, since it solves the problems for Wincdows+Chrome, a more common combo.

I think we need to use `ctrl+shift` if we want it to work the same on all OS and browsers.

**Comment by neilfranci** [comment]
Does `ctrl+shift+F` or `ctrl+shift+H` work on Firefox?
F stands for Fresh would make sense.

**Comment by aklinker1** [comment]
On mac, cmd+shift+F toggles fullscreen for a window, and `ctrl+shift+H` opens a history panel lol

**Comment by aklinker1** [comment]
What about `ctrl+;`? Doesn't stand for anything, but it might be unique?

**Comment by aklinker1** [comment]
Ugh
<img width="664" alt="Screenshot 2024-01-12 at 8 28 00 AM" src="https://github.com/wxt-dev/wxt/assets/10101283/471499a4-5904-4fe0-9d4e-3eb70af0de0b">

**Comment by neilfranci** [comment]
Why does Mac have so many shortcuts? lol. It would work and finger-friendly. if that `check document` does nothing seriously...

**Comment by aklinker1** [comment]
RIP doesn't like non-alphanumerics...

<img width="317" alt="Screenshot 2024-01-12 at 8 32 03 AM" src="https://github.com/wxt-dev/wxt/assets/10101283/aacc8bbe-2e3d-414c-9ff3-f81d3657292f">

**Comment by aklinker1** [comment]
Wait, your original suggestion of `Alt+E` might work. Works on mac, for Firefox and Chrome. Can you check it on windows?

**Comment by neilfranci** [comment]
nah, it open the menu setting

![image](https://github.com/wxt-dev/wxt/assets/66520542/1f4a1280-a0b4-4acf-9b45-c5b23145267e)

Maybe `Alt+B` ? worked for me

**Comment by aklinker1** [comment]
Alt+B works, but it's a little far from the option key on my keyboard. What about `Alt+R`?

**Comment by neilfranci** [comment]
Seem worked for me.

**Comment by aklinker1** [comment]
Alright, I'll update it to `Alt+R`! That was a nightmare lol

**Comment by neilfranci** [comment]
np, glad to help!

**Comment by aklinker1** [comment]
Released in [`v0.14.1`](https://github.com/wxt-dev/wxt/releases/tag/v0.14.1]
